rozwiązanie 1

iordakowska opened the pull request:
Póki co tyle udało mi się zrobić.

iordakowska committed:
rozwiązanie 1

slawekludwiczak commented on src/main/resources/templates/done.html:
```diff
@@ -0,0 +1,25 @@
+    <meta charset="UTF-8">
+    <title>archiwum</title>
+
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js" integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM" crossorigin="anonymous"></script>
```
Te elementy powtarzają się w szablonach, postaraj się je też wrzucić do fragmentów

slawekludwiczak commented on src/main/resources/templates/done.html:
```diff
@@ -0,0 +1,25 @@
+  </ul>
+</div>
+<br>
+<a href="/">Powrót do strony głównej</a><br>
```
Wyśrodkuj to tak, jak pozostałe elementy strony

slawekludwiczak commented on src/main/resources/templates/home.html:
```diff
@@ -0,0 +1,31 @@
+    </ul>
+</div>
+
+<h4>Lista zadań</h4>
+<a href="/ready">do wykonania</a><br>
+<a href="/done">archiwum</a>
```
Wyśrodkuj to tak, jak pozostałe elementy strony

slawekludwiczak commented on src/main/resources/templates/ready.html:
```diff
@@ -0,0 +1,25 @@
+  </ul>
+</div>
+<br>
+<a href="/">Powrót do strony głównej</a><br>
```
Wyśrodkuj to tak, jak pozostałe elementy strony

slawekludwiczak reviewed (commented):
Jeżeli definiujesz style, to najlepiej definiuj klasy i używaj zewnętrznego arkusza css. W takiej prostej aplikacji używanie atrybutu style jest jeszcze akceptowalne, ale nie warto się do tego przyzwyczajać :)

Punktacja 
wyświetlanie listy zadań do zrobienia i dodawanie zadań - 0.5 / 1 pkt
odznaczanie gotowego zadania - 0 / 1 pkt
wyświetlanie archiwum - 1 / 1 pkt
dwie dodatkowe funkcjonalności (wybrane przez siebie z listy powyżej) - 2 / 2 pkt
szata graficzna - 0.8 / 1 pkt
Łącznie 4.3 / 6

https://youtu.be/hBQVqWN2rco

iordakowska committed:
rozwiązanie 2
Dodanie zadania:
jak ustawić value w select form.html - enum status, category
jak dodac pole z datą

iordakowska committed:
rozwiązanie 2
Dodanie zadania:
jak ustawić value w select form.html - enum status, category
jak dodac pole z datą

slawekludwiczak commented:
https://youtu.be/DXt9swx2qXw

iordakowska committed:
rozwiązanie 3
Dodawanie taska
Użytkownik pojawił się ponieważ temat "Spring data" maglowałam
